substrate/runtime-support/src/lib.rs

```diff
@@ -22,18 +22,17 @@ extern crate substrate_runtime_std as rstd;
 extern crate substrate_runtime_io as runtime_io;
 extern crate substrate_codec as codec;
 extern crate substrate_primitives as primitives;
-#[macro_use]
-#[cfg(any(feature = "std", test))]
-extern crate hex_literal;
+#[macro_use] #[cfg(any(feature = "std", test))] extern crate hex_literal;
+#[cfg(feature = "std")] extern crate ed25519;
 
 pub mod storage;
 mod hashable;
 #[macro_use]
 #[cfg(feature = "std")]
-mod testing;
+mod keyring;
 
 pub use self::storage::StorageVec;
 pub use self::hashable::Hashable;
 
 #[cfg(feature = "std")]
-pub use self::testing::{one, two};
+pub use self::keyring::Keyring;
```

Comment:
when this is re-exported, it's not clear that it's for testing.

Reply:
well... could export only with testing config, i guess..

Reply:
actually that won't work

Reply:
yeah, sadly compiling with `cfg(test)` doesn't carry down to the dependencies. Why not extract it out to a separate library? I don't see any reason it should live in `runtime-support` when it's more of a testing utilities kind of thing.